src/tests/ftest/launch.py

```diff
@@ -1593,19 +1593,25 @@ def install_debuginfos():
 
     if USE_DEBUGINFO_INSTALL:
         distro_info = detect()
-        yum_args = ["--exclude", "ompi-debuginfo", "openmpi3"]
+        dnf_args = ["--exclude", "ompi-debuginfo", "openmpi3"]
         if "suse" in distro_info.name.lower():
-            yum_args.extend(["libpmemobj1", "python3"])
+            dnf_args.extend(["libpmemobj1", "python3"])
+            # There are no "debug" repos
+            # Packages are found within daos-stack* repos and opensuse* repos
+            cmds.append(
+                ["sudo", "dnf", "debuginfo-install", "--enablerepo='*'", "-y"] +
+                dnf_args + ["daos-server", "gcc"])
         elif "centos" in distro_info.name.lower():
-            yum_args.extend(["libpmemobj", "python36"])
+            dnf_args.extend(["libpmemobj", "python36"])
+            cmds.append(
+                ["sudo", "debuginfo-install", "--enablerepo=*-debuginfo", "-y"] +
```

Comment:
(pylint-line-too-long) Line too long (81/80)

Reply:
@omar-ocampo : Looks like you have to fix this check-patch issue.

Reply:
:(